src/api/queues.js

```diff
@@ -230,6 +230,30 @@ router.patch(
   })
 )
 
+// Add a starred queue
+router.post(
+  '/:queueId/star/:userId',
```

Comment:
Allowing the user ID to be specified by the client without any constraint checking on the server means that any user can star any queue for any other user. A better way of implementing this might be to use the identity of making the request.